### ISSUE TITLE
Cannot see chats

### ISSUE THREAD

**Issue by ECon87** [normen/whatscli#56]
Initially it fails to login thrown an admin 401 error. Then, it connects but does not show my active chats.
See attached picture.

Thanks.

![Screenshot_2021-11-04_10-24-56](https://user-images.githubusercontent.com/46789268/140355139-c84036b5-90cd-469f-a65f-6f34fb431144.png)

**Comment by gglanzani** [comment]
Also seeing this since yesterday (without the (admin 401 error first I believe)!

**Comment by normen** [comment]
Tre to logout, then log back in. Also make sure you use the latest version 1.0.11

**Comment by ECon87** [comment]
I am trying to log back in with a QR Code but i a get panic message

**Comment by normen** [comment]
Did you `/logout` before? "Panic message" isn't very specific..

**Comment by ECon87** [comment]
yes i did logout. Yeah I know, but it failed and through the error quickly. I will try again

**Comment by normen** [comment]
see #51
